proposal/comp110_architecture_proposal.tex

```diff
@@ -6,72 +6,131 @@
+	\end{description}
+	
+	\section*{Paper 3}
+	\begin{description}
+		\item[Title:] Dynamically adjusting game-play in 2D Platformers using
+		Procedural Level Generation
+		\item[Citation:] \cite{dynamic}
+		\item[Abstract:] ''The rapid growth of the entertainment industry has presented the requirement for more efficient
+		development of computerized games. Importantly, the diversity of audiences that participate in
+		playing games has called for the development of new technologies that allow games to address
+		users with differing levels of skills and preferences. This research presents a systematic study that
+		explored the concept of dynamic difficulty using procedural level generation with interactive
+		evolutionary computation. Additionally, the design, development and trial of computerized agents
+		the play game levels in the place of a human player is detailed. The work presented in this thesis
+		provides a solution to the rapid growth of the entertainment industry whilst providing a more
+		effective means for developing computerized games''
+		\item[Web link:] \url{http://ro.ecu.edu.au/cgi/viewcontent.cgi?article=1095&context=theses_hons}
+		\item[Full text link:] \url{http://ro.ecu.edu.au/cgi/viewcontent.cgi?article=1095&context=theses_hons}
+		\item[Comments:] By just reading the abstract alone I found this one to be greatly interesting and relative to the point at hand. This was found through the internet and key word searches.
+	\end{description}
+	
+	\section*{Paper 4}
+	\begin{description}
+		\item[Title:] Rhythm-Based Level Generation for 2D Platformers
+		\item[Citation:] \cite{Rhytem}
+		\item[Abstract:] ''We present a rhythm-based method for the automatic generation
+		of levels for 2D platformers, where the rhythm is that which the
+		player feels with his hands while playing. Levels are created using
+		a grammar-based method: first generating rhythms, then
+		generating geometry based on those rhythms. Generation is
+		constrained by a set of style parameters tweakable by a human
+		designer. The approach also minimizes the amount of content that
+		must be manually authored, instead relying on geometry
+		components that are included in the level designer’s tileset and a
+		set of jump types. Our results show that this method produces an
+		impressive variety of levels, all of which are fully playable. ''
+		
+		\item[Web link:] \url{https://users.soe.ucsc.edu/~ejw/papers/smith-platformer-generation-fdg2009.pdf}
+		\item[Full text link:] \url{https://users.soe.ucsc.edu/~ejw/papers/smith-platformer-generation-fdg2009.pdf}
+		\item[Comments:] Rhythm-Based Level Generation for 2D Platformers is an interesting concept on 2D platform creation and how it produces a variety of levels, it is also cited by over 70 people. This was found through the internet and key word searches.
+	\end{description}
+	
+	\section*{Paper 5}
+	\begin{description}
+		\item[Title:] Level Generation System for Platform Games
```

Comment:
This is an unpublished thesis. Look to see if the author has published something in the academic peer-review literature.